picard/ui/infodialog.py

```diff
@@ -296,6 +297,20 @@ def _display_info_tab(self):
             tabWidget.setTabText(tab_index, _("&Info"))
             self.tab_hide(tab)
 
+class TrackInfoDialog(InfoDialog):
```

Comment:
hmm, I tested it with zas' example release with 3 images (e5e80fb7-f6b7-3c51-9183-5af44a6c6195) and there it works, but you're right, for single image albums it's not accesible. I'll try to fix that